components/Navigation/NavigationLogin.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <ul style="list-style: none;">
+    <li v-if="loginStatus">
+      <nuxt-link to="/user/logout">Logout</nuxt-link>
+    </li>
+    <li v-else>
+      <nuxt-link to="/user/login?destination=/">Login</nuxt-link>
+    </li>
+  </ul>
+</template>
+
+<script setup>
+const loginStatus = ref(false)
+
+onMounted(async () => {
```

Comment:
instead of re-rendering on-mounted, let's better make this a client-only component and only render on the client + keep the code clean